Sources/DistributedSystem/SyncCallManager.swift

```diff
@@ -6,64 +6,134 @@
 //
 // http://www.apache.org/licenses/LICENSE-2.0
 
+import Atomics
 import PackageConcurrencyHelpers
 import DistributedSystemConformance
+import Helpers
+import Logging
 internal import NIOCore
 
 class SyncCallManager {
+    private let loggerBox: Box<Logger>
+    private var nextID = ManagedAtomic<UInt64>(0)
     private var lock = Lock()
-    private var nextID: UInt64 = 1
-    private var continuations: [UInt64: CheckedContinuation<any DistributedSystem.SerializationRequirement, Error>] = [:]
+    private var continuations = [UInt64: CheckedContinuation<any DistributedSystem.SerializationRequirement, Error>]()
```

Comment:
Alternative is to use single `@Protected` dictionary `CallID` -> `Either<ByteBuffer, Continuation>`.

Reply:
Yes, but what is a purpose of such change?
`results` will be used in a really rare number of cases when result came sooner than waiting thread suspended on continuation...